### PR TITLE
feat(60): AWS creating task

### PR DESCRIPTION
**Pull request by ArtemGermanenko**

### CODE REVIEW

File: server/aws-middleware/file-api.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const idOutputs = [];
+
+  for (let i = 0; i < length;) {
+    uploadInput(file['input' + ++i], idBucket, i);
+    const _id = new mongoose.Types.ObjectId();
+    idInputs.push(_id);
+    taskApi.addFile({
+      _id,
+      name: `input${i}`,
+      url: `Task_${idBucket}/Inputs/input${i}`,
```
Comment (slawiko) [inline server/aws-middleware/file-api.js]: This string template must be extract into function.

What if you have to change S3 bucket structure?

File: server/initialization/aws.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const AWS = require('aws-sdk');
+
+const credentials = new AWS.SharedIniFileCredentials({ profile: 'artem' });
```
Comment (slawiko) [inline server/initialization/aws.js]: Code in repository must be unpersonalized

File: server/aws-middleware/file-api.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  });
+};
+
+awsModule.createTask = function (file, input, length) {
+  const idBucket = new mongoose.Types.ObjectId();
+  const idInputs = [];
+  const idOutputs = [];
+
+  for (let i = 0; i < length;) {
+    uploadInput(file['input' + ++i], idBucket, i);
```
Comment (slawiko) [inline server/aws-middleware/file-api.js]: I do not understand how it works. Did you forget that it is async code?

Looks like even if `uploadInput` fails, you'll try to `taskApi.addFile` and `taskApi.addTask`. You do not wait until any request finishing. WTF

Maybe I'm wrong, so just for misunderstanding exclusion: 
`taskApi.addFile`, `awsModule.upload`, `taskApi.addTask` are these methods async?

**Comment by ArtemGermanenko** [comment]
@slawiko Done

**Pull request merged**